CHANGES.txt

```diff
@@ -6,6 +6,9 @@ Changes
 
 Bug fixes:
 
+- The flag for GDAL driver registration has been changed to an _env module
+  attribute. Drivers should only be registered once per process at most
+  (#3260).
```

Comment:
```suggestion
  (#3250).
```

Reply:
@Multihuntr @vincentsarago Since it's a change log I've generally linked the PR that makes the change.

Reply:
oh sorry I'll revert this